LICENSE

```diff
@@ -215,6 +215,7 @@ Apache 2.0 licenses
 The following components are provided under the Apache License. See project link for details.
 The text of each license is the standard Apache 2.0 license.
 
+prometheus(common) v0.15.0 https://github.com/prometheus/common Apache 2.0 licenses
```

Comment:
This is source code LICENSE, you should update https://github.com/apache/skywalking-satellite/blob/main/dist/LICENSE.

Reply:
OK